docs/sources/administration/service-accounts/create-service-account.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Create a service account in Grafana
+menuTitle: Create a service account
+description: 'How to create a service account in Grafana'
+weight: 50
+keywords:
+  - Service accounts
+---
+
+# Create a service account in Grafana
+
+A service account is a user account that can be used to run automated or compute workloads. For more information, refer to [About service accounts]({{< relref "../service-accounts/about-service-accounts.md#">}}).
+
+For more information about creating service accounts via the API, refer to [Create service account via API]({{< relref "../../http_api/serviceaccount.md#create-service-account">}}).
+
+Before you begin:
+
+- Ensure you have added the feature toggle for service accounts `service-accounts`. [Enable service accounts]({{< relref "./enable-service-accounts.md#">}})
+- Ensure you have permission to create and edit service accounts, for more information refer to [About users and permissions]({{< relref "../manage-users-and-permissions/about-users-and-permissions.md#">}})
+
+**To create a service account:**
+
+1. Sign in to Grafana and hover your cursor over the organization icon in the sidebar.
+1. Click **Service accounts**.
+1. Click **New service account**.
+1. Enter a display name.
```

Comment:
The formatting is broken here. I'd recommend to split Display Name docs to a separate section to be honest, there are constraints and patterns we suggest, so it's worth to  have it separate. What do you think?

<img width="1095" alt="Screenshot 2022-04-07 at 12 43 00" src="https://user-images.githubusercontent.com/1861190/162181717-2137e473-d189-4bf5-890c-23ca0253e9eb.png">